weave-js/src/components/Form/Select.tsx

```diff
@@ -178,12 +204,13 @@ const getStyles = <
         ? hexToRGB(RED_550, 0.64)
         : hexToRGB(TEAL_500, 0.64);
       const height = HEIGHTS[size];
+      const minHeight = MIN_HEIGHTS[size] ?? height;
```

Comment:
whats this default for? wouldn't minHeight always have a value if height has a value?

Reply:
`MIN_HEIGHTS` has `undefined` as the value for most sizes, e.g. `MIN_HEIGHTS['small']` is undefined, while `HEIGHTS['small']` is 24.